packages/expo-notifications/src/getExpoPushTokenAsync.ts

```diff
@@ -60,9 +74,9 @@ export default async function getExpoPushTokenAsync(options: Options = {}): Prom
     type,
     deviceId: deviceId.toLowerCase(),
     development,
-    experienceId,
     appId: applicationId,
     deviceToken: getDeviceToken(devicePushToken),
+    ...experienceIdOrProjectIdBodyParam,
```

Comment:
Could inline it for brevity without loss of clarity (imo)
```suggestion
    ...projectId ? { projectId } : { experienceId },
```